db/migrate/20160531170531_remove_duplicate_aspect_visibilities.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveDuplicateAspectVisibilities < ActiveRecord::Migration
+  def up
+    where = "WHERE a1.aspect_id = a2.aspect_id AND a1.shareable_id = a2.shareable_id AND "\
+      "a1.shareable_type = a2.shareable_type AND a1.id > a2.id"
+    if AppConfig.postgres?
+      AspectVisibility.connection.execute(
```

Comment:
you can use `execute` directly without `AspectVisibility.connection.` in migrations :smiley: